src/app.ts

```diff
@@ -122,4 +184,135 @@ app.get(
   }
 )
 
+app.get(
+  '/joinGame',
+  celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      playerToken: Joi.string().required().min(1).max(250).label('playerToken'),
+      nickname: Joi.string().required().max(20).label('nickname'),
+      gameId: Joi.number().required().min(0).max(999999).label('gameId'),
+    }),
+  }),
+  async (req, res) => {
+    let correctToken = true
+    if (process.env.JEST_WORKER_ID === undefined) {
```

Comment:
I think it would remove some LOC and make join game and create game functions  if we made `verifyFCMToken` function always return true when testing, instead of not calling the function when testing and checking for it in every function.

Reply:
Done.